### PR TITLE
feat: Add a CodeZero Desktop app tutorial

### PR DESCRIPTION
**Pull request by Canuckaholic**
## Description

Used this for the UX user testing session.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Technical improvement or removal of technical debt. 
- [ ] Minor change: comments, documentation, trivial fixes

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- [ ] Unit Tests

Optional:

- [ ] Functional Tests
- [ ] Integration Tests

### CODE REVIEW

File: content/tutorials/desktop-app.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+A Workspace is a folder on your machine where you keep all your [Development Profiles](/concepts/profiles). Reminder that a Development Profile is a simple manifest file that defines a specific set of reproducible CodeZero commands. This allows developers to easily get several commands running for the tasks at hand without having to remember all the command line parameters for the command line tool.
+
+Click on the tray icon again and select Workspaces -> + Add Workspace. Select the root of the Sample Project repo.
+
+### Run a Development Profile
+
+The Sample Project comes with an example Development Profile with just a single Teleport command:
+
+```yaml
+apiVersion: system.codezero.io/v1alpha1
```
Comment (conneryn) [inline content/tutorials/desktop-app.md:99]: I'm not sure if we should highlight the actual contents of the development profile.

See proposed alternative text, or, we could show the CLI equivalent if we wanted?
`czctl teleport deployment sample-project-core -n sample-project -f ./env.sh`

File: content/tutorials/desktop-app.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+### Add a Workspace
+
+A Workspace is a folder on your machine where you keep all your [Development Profiles](/concepts/profiles). Reminder that a Development Profile is a simple manifest file that defines a specific set of reproducible CodeZero commands. This allows developers to easily get several commands running for the tasks at hand without having to remember all the command line parameters for the command line tool.
+
+Click on the tray icon again and select Workspaces -> + Add Workspace. Select the root of the Sample Project repo.
+
+### Run a Development Profile
+
+The Sample Project comes with an example Development Profile with just a single Teleport command:
```
Comment (conneryn) [inline content/tutorials/desktop-app.md:96]:
```suggestion
The Sample Project comes with an example Development Profile called "teleport", which opens a basic teleport session to the `sample-project` namespace and exports the `sample-project-core` deployment's environment variables to the file `./env.sh`.
```

File: content/tutorials/desktop-app.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+You will then need to obtain the appropriate ingress service IP address or the LoadBalancer IP address and go to `http://IP-ADDRESS` in a browser.
+
+```bash
+kubectl get svc -n sample-project sample-project-frontend --output jsonpath='{.status.loadBalancer.ingress[0].ip}'
+```
+
+### Run a Service Locally
+
+```bash
+yarn start-frontend
```
[the remark this answers is not in the document]
Comment (Canuckaholic) [reply content/tutorials/desktop-app.md:71]: I'm open to however we want to structure this. So perhaps this tutorial starts with something like "this tutorial assumes you have completed tutorials #1 and #2"?

File: content/tutorials/desktop-app.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+If you have already installed the Sample Project to a cluster, you can skip to the next section. If not, head on over to the [GitHub repo](https://github.com/c6o/sample-project) and go through the steps in the Sample Project's README.
+
+### Run a Service Locally
+
+If you haven't already, go through the [Developing Edge Services](https://docs.codezero.io/#/tutorials/edge) tutorial, which will show you how to run a teleport command using the CodeZero CLI to debug a locally running service. Close the teleport at the end because we are going to run it again in this tutorial, but using the Desktop app.
+
+### Install the Desktop app
+
+Head over to the [CodeZero website](https://codezero.io/platform/desktop) and download the Desktop app for your platform. Run the installer.
```
Comment (nsainaney) [inline content/tutorials/desktop-app.md:39]: This should be under Guides and by the time you come to the Tutorial, you can assume they have it installed

File: content/tutorials/desktop-app.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+[prerequisites](_fragments/prerequisites.md ':include')
+
+If you are missing any of these, you can [learn about getting a Kubernetes cluster](guides/kubernetes-quickstart), clone the [Sample Kubernetes Project](https://github.com/c6o/sample-project) on GitHub, and follow this guide for [installing the CodeZero CLI and Deskop app](/guides/installing).
+
+For this tutorial you will also need to be using MacOS or Linux, and have NodeJS 16+.
+
+## Tutorial
+
+### Set up the Sample Project
```
Comment (nsainaney) [inline content/tutorials/desktop-app.md:29]: Make this document consistent with https://docs.codezero.io/#/tutorials/edge

See how Prerequisites are listed and summarized. Set up a Sample Project is a prerequisite and should not be repeated in each tutorial unless there are changes to the setup that need to be highlighted

**Review by nsainaney — CHANGES_REQUESTED** [review]

**Review by conneryn — APPROVED** [review]

**Review by nsainaney — APPROVED** [review]

**Pull request merged**